prow/config/jobs/envoy.yaml

```diff
@@ -36,8 +36,8 @@ jobs:
 resources:
   default:
     requests:
-      memory: "50Gi"
-      cpu: "14000m"
+      memory: "110Gi"
+      cpu: "30000m"
     limits:
-      memory: "60Gi"
-      cpu: "16000m"
\ No newline at end of file
+      memory: "120Gi"
+      cpu: "32000m"
```

Comment:
Outside the scope of this PR: I am wondering if we should reconsider how we use limits. See https://mcrthr.com/kubernetes-cpu-limits (and we do have an affected kernel version - https://cloud.google.com/container-optimized-os/docs/release-notes/m69#cos-69-10895-348-0). I suspect if we just make the requests what we want to ensure jobs get, then limits are just needlessly throttling in most or all cases